<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>æ™ºèƒ½æ–‡æœ¬æå–å·¥å…·</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* å…¨å±€æ ·å¼ */
QWidget {
    background-color: #f8fafc;
    font-family: 'Microsoft YaHei', 'Segoe UI', sans-serif;
}

/* æ ‡é¢˜æ ·å¼ */
QLineEdit, QTextEdit {
    background-color: white;
    border: 2px solid #e2e8f0;
    border-radius: 8px;
    padding: 10px;
    font-size: 13px;
    selection-background-color: #4299e1;
    selection-color: white;
}

QTextEdit:focus, QLineEdit:focus {
    border-color: #4299e1;
    box-shadow: 0 0 0 3px rgba(66, 153, 225, 0.15);
}

/* æŒ‰é’®åŸºç¡€æ ·å¼ */
QPushButton {
    border: none;
    border-radius: 8px;
    font-weight: bold;
    font-size: 13px;
    color: white;
    transition: all 0.2s ease-in-out;
}

QPushButton:hover {
    transform: translateY(-2px);
    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.15);
}

QPushButton:pressed {
    transform: translateY(0px);
    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
}

/* æ ‡ç­¾æ ·å¼ */
QLabel {
    color: #2d3748;
    font-size: 13px;
    font-weight: 500;
}

/* åˆ†ç»„æ¡†æ ·å¼ */
QGroupBox {
    font-weight: bold;
    font-size: 14px;
    color: #2d3748;
    border: 2px solid #e2e8f0;
    border-radius: 10px;
    margin-top: 10px;
    padding-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 10px 0 10px;
}</string>
  </property>
  <widget class="QWidget" name="titleBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-bottom: 1px solid #e2e8f0;</string>
   </property>
   <widget class="QLabel" name="appIcon">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>ğŸ“</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="appTitle">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
font-weight: bold;
color: #2d3748;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>ç”Ÿè¯æå–å·¥å…· EUWE</string>
    </property>
   </widget>
   <widget class="QLabel" name="appSubtitle">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px;
color: #718096;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Made By Bcaid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDb">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>15</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #4299e1, stop:1 #667eea);
    font-size: 14px;
    border-radius: 20px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #3182ce, stop:1 #5a67d8);
}</string>
    </property>
    <property name="text">
     <string>ğŸ“š æ‰“å¼€ç†Ÿè¯åº“</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
font-weight: bold;
color: #2d3748;</string>
   </property>
   <property name="text">
    <string>ğŸ“„ å¾…æå–æ–‡æœ¬</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>550</width>
     <height>280</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>åœ¨æ­¤è¾“å…¥æ–‡æœ¬ï¼Œæˆ–ç‚¹å‡»ä¸‹æ–¹æŒ‰é’®å¯¼å…¥æ–‡ä»¶/å›¾ç‰‡...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="functionGroup">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>240</width>
     <height>280</height>
    </rect>
   </property>
   <property name="title">
    <string>âš¡ å¿«æ·æ“ä½œ</string>
   </property>
   <widget class="QPushButton" name="leftextract">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ed8936;
    text-align: left;
    padding-left: 20px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dd6b20;
}</string>
    </property>
    <property name="text">
     <string>ğŸ“‹ å·¦ä¾§æ–‡æœ¬æå–</string>
    </property>
   </widget>
   <widget class="QLabel" name="sepLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>90</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #a0aec0;
font-size: 11px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>â€”â€”â€”â€” å¯¼å…¥æ–¹å¼ â€”â€”â€”â€”</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="input">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>115</y>
      <width>210</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #48bb78;
    text-align: left;
    padding-left: 20px;
}
QPushButton:hover {
    background-color: #38a169;
}</string>
    </property>
    <property name="text">
     <string>ğŸ“ æ–‡ä»¶å¯¼å…¥ (PDF/DOC/TXT)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ocr">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>170</y>
      <width>210</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #9f7aea;
    text-align: left;
    padding-left: 20px;
}
QPushButton:hover {
    background-color: #805ad5;
}</string>
    </property>
    <property name="text">
     <string>ğŸ–¼ï¸ å›¾ç‰‡OCRè¯†åˆ«</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>225</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #cbd5e0;
    color: #2d3748;
    font-weight: normal;
}
QPushButton:hover {
    background-color: #a0aec0;
}</string>
    </property>
    <property name="text">
     <string>ğŸ—‘ï¸ æ¸…ç©º</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyBtn">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>225</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #cbd5e0;
    color: #2d3748;
    font-weight: normal;
}
QPushButton:hover {
    background-color: #a0aec0;
}</string>
    </property>
    <property name="text">
     <string>ğŸ“‹ å¤åˆ¶</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bottomBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>850</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border-top: 1px solid #e2e8f0;</string>
   </property>
   <widget class="QPushButton" name="help">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #4299e1;
    border-radius: 20px;
    font-size: 13px;
}
QPushButton:hover {
    background-color: #3182ce;
}</string>
    </property>
    <property name="text">
     <string>â“ å¸®åŠ©</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #667eea;
    border-radius: 20px;
    font-size: 13px;
}
QPushButton:hover {
    background-color: #5a67d8;
}</string>
    </property>
    <property name="text">
     <string>â„¹ï¸ å…³äº</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #718096;
font-size: 12px;
padding-left: 5px;</string>
    </property>
    <property name="text">
     <string>âœ¨ å°±ç»ª - å¯ä»¥å¼€å§‹æå–æ–‡æœ¬</string>
    </property>
   </widget>
   <widget class="QLabel" name="statsLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>280</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #4a5568;
font-size: 12px;
font-weight: bold;
background-color: #f7fafc;
border-radius: 8px;
padding: 8px;</string>
    </property>
    <property name="text">
     <string>ğŸ“Š å­—ç¬¦æ•°: 0 | å•è¯æ•°: 0 | è¡Œæ•°: 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="decorLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>850</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                stop:0 #4299e1, stop:0.5 #9f7aea, stop:1 #ed8936);
border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::HLine</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
